src/test/java/com/exasol/adapter/dynamodb/ExasolTestInterface.java

```diff
@@ -129,8 +139,15 @@ public void createAdapterScript() throws SQLException {
             }
             statementBuilder.append("  %jvmoption ").append(jvmOptions).append(";\n");
         }
-        // noinspection SpellCheckingInspection
-        statementBuilder.append("    %scriptclass com.exasol.adapter.RequestDispatcher;\n");
+    }
+
+    public void createUdf() throws SQLException {
+        final StringBuilder statementBuilder = new StringBuilder("CREATE OR REPLACE JAVA SET SCRIPT " + ADAPTER_SCHEMA
```

Comment:
I think you can also use `test-db-builder-java` for tests, they will be more compact. But I would suggest waiting until we release next version